source/mainnet/net/installation/previous-node-downloads.rst

```diff
@@ -0,0 +1,198 @@
+
+:orphan:
+
+.. _previous-downloads:
+
+=======================
+Previous node downloads
+=======================
+
+This topic contains downloads for previous node versions if you need to upgrade a node over multiple versions.
+
+Windows - Mainnet and Testnet
+=============================
+
+**Please be aware that you should backup your configuration, as the installer will overwrite the current configuration with a standard configuration.**
+
+`Windows 4.2.2 <https://distribution.concordium.software/windows/Signed/Node-4.2.2-0.msi>`_
+
+`Windows 4.2.1 <https://distribution.concordium.software/windows/Signed/Node-4.2.1-0.msi>`_
```

Comment:
We should not link this. This node is broken and 4.2.3 is 4.2.1 with the security fix.

Reply:
So a user can upgrade from 4.1.1 to 4.2.3 directly, skipping 4.2.1?

Reply:
Yes since 4.2.3 is really 4.2.1 with a bugfix. It does not count as a "new" release in my view.